src/xpring-client.ts

```diff
@@ -61,22 +63,23 @@ class XpringClient {
         );
       }
 
-      return balance.getDrops();
+      return BigInt(balance.getDrops());
     });
   }
 
   /**
    * Send the given amount of XRP from the source wallet to the destination address.
    *
-   * @param drops A numeric string indicating the number of drops to send.
+   * @param drops A `BigInt` representing the number of drops to send.
    * @param destination A destination address to send the drops to.
    * @param sender The wallet that XRP will be sent from and which will sign the request.
+   * @returns A promise which resolves to a string representing the hash of the submitted transaction.
    */
   public async send(
-    amount: XRPAmount,
+    amount: BigInt,
```

Comment:
Any chance we could make this `BigInt | string | number` then try to cast the amount as a BigInt? Overall I think the developer experience of being able to use types you are familiar with is an advantage.

(Maybe not number because they might not take into account the overflow)